Formula/ffmpeg@5.1.rb

```diff
@@ -0,0 +1,141 @@
+
+  livecheck do
+    url "https://ffmpeg.org/download.html"
+    regex(/href=.*?ffmpeg[._-]v?(5\.1(?:\.\d+)+)\.t/i)
+  end
+
+  keg_only :versioned_formula
+
+  depends_on "pkg-config" => :build
+  depends_on "aom"
+  depends_on "aribb24"
+  depends_on "dav1d"
+  depends_on "fontconfig"
+  depends_on "freetype"
+  depends_on "frei0r"
+  depends_on "gnutls"
+  depends_on "lame"
+  depends_on "libass"
+  depends_on "libbluray"
+  depends_on "librist"
+  depends_on "libsoxr"
+  depends_on "libvidstab"
+  depends_on "libvmaf"
+  depends_on "libvorbis"
+  depends_on "libvpx"
+  depends_on "opencore-amr"
+  depends_on "openjpeg"
+  depends_on "opus"
+  depends_on "rav1e"
+  depends_on "rubberband"
+  depends_on "sdl2"
+  depends_on "snappy"
+  depends_on "speex"
+  depends_on "srt"
+  depends_on "svt-av1"
+  depends_on "tesseract"
+  depends_on "theora"
+  depends_on "webp"
+  depends_on "x264"
+  depends_on "x265"
+  depends_on "xvid"
+  depends_on "xz"
+  depends_on "zeromq"
+  depends_on "zimg"
+
+  uses_from_macos "bzip2"
+  uses_from_macos "libxml2"
+  uses_from_macos "zlib"
+
+  on_linux do
+    depends_on "alsa-lib"
+    depends_on "libxv"
+  end
+
+  on_intel do
+    depends_on "nasm" => :build
+  end
+
+  fails_with gcc: "5"
+
+  def install
+    args = %W[
+      --prefix=#{prefix}
+      --enable-shared
+      --enable-pthreads
+      --enable-version3
+      --cc=#{ENV.cc}
+      --host-cflags=#{ENV.cflags}
+      --host-ldflags=#{ENV.ldflags}
+      --enable-ffplay
+      --enable-gnutls
+      --enable-gpl
+      --enable-libaom
+      --enable-libaribb24
+      --enable-libbluray
+      --enable-libdav1d
+      --enable-libmp3lame
+      --enable-libopus
+      --enable-librav1e
+      --enable-librist
+      --enable-librubberband
+      --enable-libsnappy
+      --enable-libsrt
+      --enable-libsvtav1
+      --enable-libtesseract
+      --enable-libtheora
+      --enable-libvidstab
+      --enable-libvmaf
+      --enable-libvorbis
+      --enable-libvpx
+      --enable-libwebp
+      --enable-libx264
+      --enable-libx265
+      --enable-libxml2
+      --enable-libxvid
+      --enable-lzma
+      --enable-libfontconfig
+      --enable-libfreetype
+      --enable-frei0r
+      --enable-libass
+      --enable-libopencore-amrnb
+      --enable-libopencore-amrwb
+      --enable-libopenjpeg
+      --enable-libspeex
+      --enable-libsoxr
+      --enable-libzmq
+      --enable-libzimg
+      --disable-libjack
+      --disable-indev=jack
+    ]
+
+    # Needs corefoundation, coremedia, corevideo
+    args << "--enable-videotoolbox" if OS.mac?
+    args << "--enable-neon" if Hardware::CPU.arm?
+
+    system "./configure", *args
+    system "make", "install"
+
+    # Build and install additional FFmpeg tools
+    system "make", "alltools"
+    bin.install Dir["tools/*"].select { |f| File.executable? f }
+
+    # Fix for Non-executables that were installed to bin/
+    mv bin/"python", pkgshare/"python", force: true
```

Comment:
This didn't happen for some reason?

Reply:
It did nothing because `pkgshare` did not exist.

Reply:
Well, then I wonder why this wasn't the case for `ffmpeg`...

Reply:
> Well, then I wonder why this wasn't the case for `ffmpeg`...

Ah, because of luck. Some prior `make` step created `ffmpeg` under `pkgshare`. But not `ffmpeg@5.1`.

```
$ ls `brew --prefix ffmpeg@5.1`/share
ffmpeg man
```